src/test/regress/expected/citus_local_tables.out

```diff
@@ -333,7 +333,7 @@ ERROR:  Table 'citus_local_table_1' is a citus local table. Replicating shard of
 -- undistribute_table is supported
 BEGIN;
   SELECT undistribute_table('citus_local_table_1');
-NOTICE:  Creating a new local table for citus_local_tables_test_schema.citus_local_table_1
+NOTICE:  creating a new local table for citus_local_tables_test_schema.citus_local_table_1
 NOTICE:  Moving the data of citus_local_tables_test_schema.citus_local_table_1
```

Comment:
We'll probably need to update other notices too.
But I think this doesn't need to be part of this PR.